app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/ShowTicketActivity.java

```diff
@@ -33,13 +33,22 @@ public void onCreate(Bundle savedInstanceState) {
         moviedetailTextView = (TextView)findViewById(R.id.moviedetail);
         ticketsnumberTextView = (TextView)findViewById(R.id.ticketnumber);
         ticketqrcode = (ImageView) findViewById(R.id.ticket_qrcode);
-        //get data from last activity
+        //Get data from KinoDetailsFragment
         Intent intent = getIntent();
         String data = getIntent().getStringExtra("movie_data");
```

Comment:
As Kordian suggested in the other pull request for this activity, it would be nicer to create a model class to transfer the data here, rather than passing everything as one big string.
Also, I would split up the movieDetailTextView into several smaller ones, one for location, one for description, one for date.